### DISCUSSION TITLE
【TS/JS】打开大文件时，会卡住，有没有思路或方法可以做到像 vscode 一样丝滑

### DISCUSSION THREAD

**Discussion by souldar** [nshen/learn-neovim-lua#148] [Q&A]
打开一个经过 webpack/rollup 编译压缩的文件（几万行）时，会卡死在 initailize tsserver 阶段。有什么配置方法，可以做到像 vscode 一样丝滑的打开和 format 吗？
针对这类问题，有没有解决思路或方向呢？
目前我自己的想法是通过某种手段让 tssever 异步加载，这样可以先按纯文本打开文件。不知道是否可以呢？

**Comment by nshen** [comment]
我之前打开过一个9万行的go文件，卡住了好久，不太确定是哪里的问题。请问你是怎么确定是initailize tsserver 阶段的问题呢？

如果是lsp的问题，可以 `:LspStop` 就行了。

我这里是把 LspStop 和 treesitter 都关了还是有卡，各种插件都会影响效率

网上看到的，这样可以按文本打开 `nvim -u NONE xxx.large.file`

**Reply by souldar** [reply]
是看到右下角有 initaillize tsserver 的提示，而且会一直在那里。
我试试您提到的方法。想细致调优感觉得清楚各个插件的生效机制。。
